drivers/adxl345/adxl345.c

```diff
@@ -43,6 +44,12 @@ int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
     /* get device descriptor */
     dev->params= *params;
```

Comment:
But under further consideration, think of only copying the 'often used' fields (e.g. i2c, addr) into RAM and leave the 'use once' fields (e.g. offset) in ROM...